agent/bench-scripts/pbench-fio

```diff
@@ -758,7 +759,12 @@ function fio_run_job() {
 		fi
 	fi
 
-	echo "fio job complete"
+	if [[ ${_fio_exit_code} -eq 0 ]]; then
+		echo "fio job complete"
+	else
+		echo "fio job failed, status code ${_fio_exit_code}"
+	fi
+	return ${_fio_exit_code}
```

Comment:
Interesting that you went to the trouble above of changing an `echo` to `prinf --` and then added two new instances of `echo` down here. Is there a reason?